src/main/java/org/fcrepo/migration/foxml/FoxmlInputStreamFedoraObjectProcessor.java

```diff
@@ -468,6 +474,11 @@ private String extractInlineXml() throws XMLStreamException {
 
         private void validateInlineXml() {
             if (isInlineXml && contentDigest != null && StringUtils.isNotBlank(contentDigest.getDigest())) {
+
+                if (StringUtils.equals(contentDigest.getType(), "DISABLED")) {
+                    throw new RuntimeException("DISABLED digest. Skipping digest validation");
```

Comment:
I'm wondering if we want to throw an exception here or not. As I understood the problem initially, it seemed like we wanted to continue migration of the pids with disabled digests because it was a feature of Fedora 3. If so, we could just log this instead.

Reply:
I went the exception route to assist with building an integration test case but if you think logging would be sufficient I can revert that change.

Reply:
I think logging then return to skip the checksum validation is acceptable. For the integration test we can switch to check if the migration was successful. It looks like the `InlineXmlIT` does something similar, but for this case we can check if the resources exist in the ocfl object instead of computing checksums.

Reply:
I've now made those changes as suggested @mikejritter